### PR TITLE
Fix explanation of asset installation and CSS usage

### PR DESCRIPTION
**Pull request by dallyger**

### CODE REVIEW

**Comment by dallyger** [comment]
The current explanation for theme assets installation is confusing, so I clarified, which command does what exactly.
We had a bug, where the shop owner had run `assets:install` without `theme:compile` and all fonts were missing.
This got only recently documented and was absent before. See here: <https://github.com/shopware/docs/commit/89dfc858cdb49f21cb1566f0f497b5902263ca80>

Also, in this commit: <https://github.com/shopware/docs/commit/38d70edeab538226dba9a73610e3127ff9cb2347>
The documented path for the assets got replaced by the one, that `assets:install` overrides.
The CSS rule loads it from `theme/<theme-id>` instead.

What is the correct use here?

Twig references the bundle (plugin) asset path, while CSS references the theme asset path.

Additionally, none of the mentioned paths were correct. Sometimes the assets were in a subfolder called `assets`, sometimes in `asset` and sometimes the subfolder was missing completely.
I hope, I've fixed those too now.

Maybe someone could double-check the entire document, as I could only fix it to the best of my current knowledge.

**Review by tobiasberge — CHANGES_REQUESTED** [review]
Hi, thanks for your contribution 🎉  and sorry for my late reply.

There were a couple of changes recently to the way assets work in themes.

You can find an [ADR about the changes here](https://github.com/shopware/platform/blob/trunk/adr/2023-01-10-atomic-theme-compilation.md) as well as the [corresponding change](https://github.com/shopware/platform/commit/cd3a58499c02993267c85097b550b4d07c0fffe4).

We have read your modified article and tested it with a Theme which includes assets.

The following stuff needs adjustment in the documentation:

* Theme assets are no longer saved under `<shopware root>/public/bundles`, all theme assets are saved under `<shopware root>/public/theme`
* Theme assets (images, fonts etc.) are no longer stored inside the same folder as the compiled files (CSS and JS) to avoid duplicating assets, [see ADR "Theme Assets"](https://github.com/shopware/platform/blob/trunk/adr/2023-01-10-atomic-theme-compilation.md#theme-assets), this results in a structure like this:
    ```
    .
    └── theme
        ├── <theme-uuid>
        │   ├── css
        │   │   └── all.css
        │   └── js
        │       └── all.js
        └── <theme-asset-uuid>
            └── asset
                └── your-image.png   
    ```
* Because everything is under `<shopware root>/public/theme`, the command `bin/console assets:install` is not needed anymore and the assets will be copied during `theme:compile`
* To refernce it in the SCSS, we use the variable `$app-css-relative-asset-path`: Traversing up the directory no longer works because the assets are no longer in the same directory.
    ```scss
    body {
        background-image: url('#{$app-css-relative-asset-path}/your-image.png');
    }
    ```
* To reference it inside twig, we use `theme` as `$packageName` parameter in the `asset()` function:
    ```twig
    <img src="{{ asset('/assets/your-image.png', 'theme') }}">
    ```

File: guides/plugins/themes/add-assets-to-theme.md [hunk 1]
```diff
@@ -33,18 +33,24 @@ While working with your own theme, you might have already come across the [Theme
...
+│   ├── framework
+│   ├── administration
+│   ├── storefront
+│   └── <your-plugin-name> <-- e.g. swagbasicexampletheme
+│       └── assets <-- Your asset is copied here
+│           └── your-image.png
+└── theme
+    └── <theme-uuid>
+        └── assets <-- Your asset is copied here
+            └── your-image.png
```
Comment (tobiasberge) [inline guides/plugins/themes/add-assets-to-theme.md:53]:
```suggestion
└── theme
    ├── <theme-uuid>
    │   ├── css
    │   │   └── all.css
    │   └── js
    │       └── all.js
    └── <theme-asset-uuid>
        └── asset
            └── your-image.png   
```

File: guides/plugins/themes/add-assets-to-theme.md [hunk 2]
```diff
@@ -60,14 +66,14 @@ This way of adding custom assets refers to the default way of dealing with asset
 You can link to the asset with the twig [asset](https://symfony.com/doc/current/templates.html#linking-to-css-javascript-and-image-assets) function:
 
 ```markup
-<img src="{{ asset('bundles/swagbasicexampletheme/your-image.png', 'asset') }}">
+<img src="{{ asset('bundles/<your-plugin-name>/assets/your-image.png', 'asset') }}">
```
Comment (tobiasberge) [inline guides/plugins/themes/add-assets-to-theme.md:69]:
```suggestion
<img src="{{ asset('/assets/your-image.png', 'theme') }}">
```

File: guides/plugins/themes/add-assets-to-theme.md [hunk 3]
```diff
@@ -60,14 +66,14 @@ This way of adding custom assets refers to the default way of dealing with asset
...
 ```
 
 In SCSS you can link to the asset like the following:
 
 ```css
 body {
-    background-image: url("../asset/your-image.png");
+    background-image: url("../assets/your-image.png");
```
Comment (tobiasberge) [inline guides/plugins/themes/add-assets-to-theme.md:76]:
```suggestion
    background-image: url('#{$app-css-relative-asset-path}/your-image.png');
```

**Review by tobiasberge — APPROVED** [review]

File: guides/plugins/themes/add-assets-to-theme.md [hunk 4]
```diff
@@ -33,18 +33,22 @@ While working with your own theme, you might have already come across the [Theme
 
 {% endcode %}
 
-Next, please run the bin/console assets:install command. The command copies files from `bundlePath/Resources/public` to the `/public/bundles` directory and the assets will be copied by the `theme:compile`:
+Next, please run the command `bin/console theme:compile`. The assets from the path defined in the `theme.json` file will be copied by the `theme:compile` command to `<shopware root>/public/theme/<theme-asset-uuid>` alongside with the compiled CSS and JS which are stored in a separate folder.
```
Comment (sushmangupta) [inline guides/plugins/themes/add-assets-to-theme.md:36]:
```suggestion
Next, run the command `bin/console theme:compile`. The assets from the path defined in the `theme.json` file will be copied by the `theme:compile` command to `<shopware root>/public/theme/<theme-asset-uuid>` along with the compiled CSS and JS, which are stored in a separate folder.
```

**Review by ssltg — APPROVED** [review]

File: guides/plugins/themes/add-assets-to-theme.md [hunk 5]
```diff
@@ -60,14 +64,14 @@ This way of adding custom assets refers to the default way of dealing with asset
 You can link to the asset with the twig [asset](https://symfony.com/doc/current/templates.html#linking-to-css-javascript-and-image-assets) function:
 
 ```markup
-<img src="{{ asset('bundles/swagbasicexampletheme/your-image.png', 'asset') }}">
+<img src="{{ asset('/assets/your-image.png', 'theme') }}">
 ```
 
 In SCSS you can link to the asset like the following:
```
Comment (sushmangupta) [inline guides/plugins/themes/add-assets-to-theme.md:70]:
```suggestion
In SCSS, you can link to the asset like the following:
```

**Review by sushmangupta — CHANGES_REQUESTED** [review]

**Review by sushmangupta — APPROVED** [review]

**Pull request merged**